script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -1541,25 +1547,25 @@ So, if the child who is Roa were to die while Roa himself still lives, it would
 -- Page 228, Offset 33834.
 -- 「この世界はわずかな綻びがあれば、世界が世界自身のためにその綻びを修復します。
 "If ever a crack forms in the world,
- the world will patch it over in order to preserve itself.
+ the world will plaster it over in order to preserve itself.
```

Comment:
I think this could use a general rephrasing honestly. Structure and all. That said, I sort of came up with a more "generic" sounding metaphor, which I think is more generally understandable.
>In order to preserve itself, the world mends even the slightest of defects that appear in itself.

Reply:
I dunno, I feel like that reads a bit dry, and the original doesn't strike me as hard to grasp

Reply:
It just feels a bit silly, is all. 綻び is a more common metaphor that English sort of lacks, so I don't think it was warranted thinking to much about that. 

That said, that last "itself" is a bit weird.
>In order to preserve itself, the world mends even the slightest of defects that appear in it.